### DISCUSSION TITLE
Unable to switch from dark mode back to light mode

### DISCUSSION THREAD

**Discussion by dxvgef** [primefaces/community#2413] [PrimeVue]
My system is in dark mode, and I have set it according to the documentation:

```javascript
darkModeSelector: '.my-app-dark'
```

But using the following code, I am unable to switch the PrimeVue v4 interface to light mode. Why is that?

```javascript
const toggleColorScheme() {
    const element = document.querySelector('html');
    element.classList.toggle('my-app-dark');
}
```

**Comment by j0rgedev** [comment]
Hello @dxvgef. Could you please share your PrimeVue setup? Meanwhile, you could try following this composable used in [Sakai Template](https://github.com/primefaces/sakai-vue/blob/master/src/layout/composables/layout.js#L52).

**Reply by dxvgef** [reply]
I have solved the previous problem, but the background color of the page always follows my system Settings to display dark color, how to change the background color of the page to the default white when switching to light color?

**Reply by j0rgedev** [reply]
Hey @dxvgef, could you please share your preset? Or are you using the default ones?

**Reply by dxvgef** [reply]
> Hey @dxvgef, could you please share your preset? Or are you using the default ones?

"I've solved it! Creating a brand new project works fine. It might be because there were conflicts between the code of version 3 and version 4. Thanks!"
